src/api/app/controllers/concerns/read_access_of_deleted.rb

```diff
@@ -1,20 +1,13 @@
-require 'api_error'
-
-module ValidationHelper
-  class InvalidPackageNameError < APIError
```

Comment:
Isn't https://github.com/openSUSE/open-build-service/blob/8422cc2a00190b2468304e3a9ae89a5ecec3a120/src/api/app/controllers/source_package_command_controller.rb#L490 raising this?

Reply:
Nope. it's raising this

https://github.com/openSUSE/open-build-service/pull/18145/files#diff-85fea887f3c393a90e49afb278d55688c4f7c505fc4ccd36a4fbe8c4dd0e47ceR9

Reply:
Right